Wikipedia/UI-V5/WMFRelatedSectionController.m

```diff
@@ -107,6 +113,21 @@ - (void)configureCell:(UICollectionViewCell*)cell
     }
 }
 
+- (UIViewController*)moreViewController {
```

Comment:
It seems all "more view controllers" should be instances of `WMFArticleListCollectionViewController`. 

In that case, returning a view controller here seems to be more flexible / generic than required. The only thing that should be different for each section is the data source - so it seems appropriate to return that instead. Then we can let the view controller can just inject the data source into the articleListCollectionVC and present it.